src/ErrorNotification.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect } from 'react';
+import cn from 'classnames';
+
+interface Props {
```

Comment:
[Not critical]
Here and in the similar places for other components:
The better name for this interface would be something like `ErrorNotificationProps`, so that it precisely describes its contents and cannot be confused with other types.